pages/_document.js

```diff
@@ -17,7 +16,6 @@ export default class MyDocument extends Document {
             {utilities}
             {markdown}
           </style>
-          <style id="emotion-static">{extractCritical(page.html).css}</style>
```

Comment:
I _don't_ believe this is needed anymore, but waiting to make sure once the Now deploy goes through. I believe it was only extracting emotion prefixed styles anyways.